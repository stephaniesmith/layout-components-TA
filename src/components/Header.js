import Component from './Component';
// import styles from './Header.css';

export default class Header extends Component {
  render() {
    const links = this.props.links.map(link => {
      return `<a href="/${link}">${link}</a>`;
    });

    return `
      <header>
        <nav>
          ${links.join(' ')}
        </nav>
      </header>`;
  }
}
