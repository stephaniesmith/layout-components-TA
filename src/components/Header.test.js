import Header from './Header';

describe('Header Component', () => {
  it('should be a header', () => {
    const header = new Header({
      links: ['about']
    });

    expect(header.render()).toMatchSnapshot();
  });
});
