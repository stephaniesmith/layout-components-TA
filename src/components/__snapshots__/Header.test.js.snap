// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should be a header 1`] = `
"
      <header>
        <nav>
          <a href=\\"/about\\">about</a>
        </nav>
      </header>"
`;
